docs/pages/index.mdx

```diff
@@ -35,18 +35,40 @@ console.log(res.rows[0].message) // Hello world!
 await client.end()
 ```
 
-And here's the same thing with callbacks:
+### Error Handling
 
-```js
-const { Client } = require('pg')
+For the sake of simplicity, these docs will assume that the methods are successful. In real life use, make sure to properly handle errors thrown in the methods. A `try/catch` block is a great way to do so:
+
+```ts
+import { Client } from 'pg'
 const client = new Client()
+await client.connect()
+
+try {
+   const res = await client.query('SELECT $1::text as message', ['Hello world!'])
```

Comment:
3-space indents and a semicolon appear after this point

Reply:
Hooray for my VS Code settings bleeding through due to the lack of an `.editorconfig` 😬 I'll tweak it 👍🏻